src/domain/transfer/dto.js

```diff
@@ -101,6 +139,13 @@ const producerConfigDto = (functionality, action, logPrefix = '') => {
   return Object.freeze({ topicConfig, kafkaConfig })
 }
 
+const setOriginalRequestPayload = async (kafkaMessageContext, payloadCache = undefined) => {
+  await payloadCache.setPayload(
+    kafkaMessageContext.originalRequestId,
+    kafkaMessageContext.originalRequestPayload
```

Comment:
Please make sure that we are not passing payload in kafka context if this redis payload cache mode is set.

Reply:
- https://github.com/mojaloop/ml-api-adapter/pull/546/files#diff-2a1ae61fd420ab3f7aa02aa844e47009053e271008c0e46c1dfed6421985368cR79
- https://github.com/mojaloop/ml-api-adapter/pull/546/files#diff-2a1ae61fd420ab3f7aa02aa844e47009053e271008c0e46c1dfed6421985368cR150
- https://github.com/mojaloop/ml-api-adapter/pull/546/files#diff-2a1ae61fd420ab3f7aa02aa844e47009053e271008c0e46c1dfed6421985368cR263